### ISSUE TITLE
Color contrast ratio of the keyboard focus present on menu items present inside "Azure function" control is less than 3:1: A11y_Azure Tools for VS Code_Azure Function_Create New Project_Non-textContrast

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-azurefunctions#4965]
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.**
**GitHubTags:**
#A11yMAS;#A11yTCS;#A11ySev2;#E2E_AzureToolsforVSCode_Win32_April2026;#Azure Tools for VS Code;#Win32;#FTP;#DesktopApp;#WCAG1.4.11;#Non-textContrast;

### Environment Details:
Application Name: Azure Tools for VS Code
Application Version: 1.114.0 (User Setup)
Operating System: Windows 11 version 25H2

### Repro Steps:

1. Launch Visual Studio Code.
2. TAB till extension button and search for "Azure Functions".
3. Press ENTER on install and tab to the "Azure" button.
4. Tab till 'Azure Functions' button and hit enter.
5. Tab till 'Create new project' button.
6. Open accessibility insights for windows and check the color contrast ratio of keyboard focus present on the menu items.
7. Observe the issue Color contrast ratio of the keyboard focus present on menu items present inside "Azure function" control is less than 3:1

Actual:
Color contrast ratio of the keyboard focus present on menu items present inside "Azure function" control is less than 3:1.

Expected:
Color contrast ratio of the keyboard focus present on menu items present inside "Azure function" control should be greater than or equals to 3:1

User Impact:
Low vision users will not be able to see the keyboard focus properly if color contrast ratio of it will be less than 3:1.

Attachment:

<img width="1270" height="638" alt="Image" src="https://github.com/user-attachments/assets/0a0b4056-0778-4859-98a5-2ae1a28f7fe1" />

**Comment by bwateratmsft** [comment]
This is ultimately a VSCode issue, not us.

**Comment by jinglouMSFT** [comment]
Do we have the bug ID in VS Code we are tracking?

**Comment by MicroFish91** [comment]
Reached out offline as well, but please provide the VS Code color theme being used to repro this

**Comment by kapilvaishna** [comment]
External bug: https://github.com/microsoft/vscode/issues/314929
Once external bug is fixed and close we will verify and close this bug.

**Comment by bwateratmsft** [comment]
The upstream issue has been closed as fixed in 1.124.0: https://github.com/microsoft/vscode/issues/314929

**Comment by MicroFish91** [comment]
@kapilvaishna Could you please mark this as resolved? Thank you

**Comment by kapilvaishna** [comment]
Issue is not repro in VS code insider 1.124.0 In Light 2026 and dark 2026 theme so closing the bug.

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/5186c7b4-01f0-4200-9397-8549072ee058" />


<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/5c1fb300-87c5-495d-bbd5-9570b2926f67" />


<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/8f5ef9b2-3835-40d5-9e4e-650a5d303cce" />

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/b8d9d270-d235-4fda-bc60-3624bac918f7" />
